### DISCUSSION TITLE
CSV uzantılı dosyalar

### DISCUSSION THREAD

**Discussion by btelliNku** [gusanmaz/BMB212-Algorithms-2223#3] [Q&A] [answered]
Uygulamalar yalnızca CSV dosyaları ile çalıştırılacak değil mi ?

**Comment by gusanmaz** [comment, marked as answer]
Ilk 6 odev icin evet. Ilk 5 odevde yazacaginiz programlar ogrenci bilgileri iceren CSV dosyalari kullanilarak test edilecek. 6. odevde ise yazacaginiz program path bilgilerini iceren CSV dosyalari ile test edilecek.
